packages/access-api/src/types/provisions.ts

```diff
@@ -0,0 +1,30 @@
+import * as Ucanto from '@ucanto/interface'
+
+export type AlphaStorageProvider = 'did:web:web3.storage:providers:w3up-alpha'
+
+/**
+ * action which results in provisionment of a space consuming a storage provider
+ */
+export interface StorageProvisionCreation {
```

Comment:
Nit: I find naming to be very confusing, can we simply call this

```suggestion
export interface ProvisionModel {
```

Or better yet

```suggestion
export interface Provision {
```

Reply:
https://github.com/web3-storage/w3protocol/pull/462/commits/c1e37f9408c2f75527b01e57fdf526cd2fe2d2fc